src/indicators/wfp_market_monitor/utils/raw_market_monitor.R

```diff
@@ -0,0 +1,13 @@
+box::use(cs = ../../../../src/utils/cloud_storage)
+
+#' Download raw market monitoring data
+#'
+#' Downloads raw market monitoring data from Azure database. Is uploaded directly
+#'
+#' @export
+raw <- function() {
```

Comment:
Could you clarify how the overall monitoring process will work for this data since we're referencing static inputs? How often will WFP send new data? Do we want to have validation that any new CSV files have the same structure as expected?

Reply:
Yes, good idea to have the test! They are updating whenever they have updates, currently maybe every other week, but need to get confirmation of them that this pipeline is going to be frequently updated and monitored. THye are by far the most difficult data provider.

Reply:
Currently just checking that at least the columns we need exist.